pkg/util/constants/constant_darwin.go

```diff
@@ -0,0 +1,23 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
I'm not sure that `constants` nor the full import path is descriptive enough name to tell what is contained within it. What about `pkg/common/paths/*` or similar?